src/lib/components/icons/IconTool.svelte

```diff
@@ -0,0 +1,10 @@
+<script lang="ts">
+	export let classNames = "";
+</script>
+
+<svg width="16" height="16" class={classNames} fill="none" xmlns="http://www.w3.org/2000/svg"
```

Comment:
like in https://github.com/huggingface/chat-ui/blob/dbc0ca76f290ad222f93da5dc7beb3966c011cfc/src/lib/components/icons/IconNew.svelte#L8-L11

we usually use `width="1em" height="1em" viewBox="0 0 32 32"` 

`viewBox="0 0 32 32"` part might be different depending on the icon